packages/emoji-mart/src/components/Picker/Picker.tsx

```diff
@@ -702,7 +702,10 @@ export default class Picker extends Component {
                 </div>
               </div>
             ) : (
-              <div class="color-c" style={{ fontSize: 21 }}>
+              <div
+                class="color-c"
+                style={{ fontSize: this.props.emojiPreviewSize }}
```

Comment:
I would recommend renaming this to something more clear, like `previewTextFontSize`.